website_product_document_download_counter/readme/DESCRIPTION.md

```
@@ -0,0 +1,9 @@
+Description
+========================================
+
+This module allows you to track how many times product-related documents
+are downloaded from the Odoo website.
+
+Main features:
+- Automatic logging of downloads for documents attached to products.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)